basestation/bot/bot_manager.py

```diff
@@ -73,8 +72,7 @@ def add_simbot(self, id: int, angle: int, x: int, y: int, size: int):
         """
         name = self.__sim_names[randint(0, len(self.__sim_names))]
         sim_bot = SimBot(id, angle, x, y, size)
-        print(sim_bot)
-        self.__bot_map[name] = sim_bot
+        self.__bot_map["SIM " + name] = sim_bot
```

Comment:
I would add a comment here on why we add "SIM" to the name

Reply:
done!